### PR TITLE
Updates to support VMWare variant of Hypervisor for MacOS Build Nodes

### PR DESCRIPTION
**Pull request by nycnewman**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [x] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

CHANGELOG_BEGIN
Add VMWare documentation for MacOS Build Nodes
CHANGELOG_END

### CODE REVIEW

File: infra/macos/scripts/rebuild-crontask.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (garyverhaegen-da) [inline infra/macos/scripts/rebuild-crontask.sh:1]:
```suggestion
# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
#!/bin/bash
```

File: infra/macos/scripts/run-agent.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (garyverhaegen-da) [inline infra/macos/scripts/run-agent.sh:1]:
```suggestion
# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
#!/bin/bash
```

File: infra/macos/scripts/rebuild-crontask.sh (hunk 1 above)
Comment (SamirTalwar-DA) [reply infra/macos/scripts/rebuild-crontask.sh:1]: I think the shebang has to go on the first line.

In _build.sh_, we use this:

```suggestion
#!/usr/bin/env bash
# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
# SPDX-License-Identifier: Apache-2.0
```

(Probably meaningful that we use `/usr/bin/env bash`, too. Mine, at least, comes from `$HOME/.nix-profile/bin/bash`.)

Comment (garyverhaegen-da) [reply infra/macos/scripts/rebuild-crontask.sh:1]: At least on my machine, having a comment before the shebang works. Not sure if this is a macOS thing.

Re: `env`, yes, this is usually the preferred way to invoke bash when writing portable scripts. In this case, though, this script is meant to run within a VM we completely control (created by the rest of this folder), so I think using `/bin/bash` is fine.

Comment (garyverhaegen-da) [reply infra/macos/scripts/rebuild-crontask.sh:1]: Silly me. Not fully awake yet it seems. Corrected shebang lines to be first.

**Pull request merged**